src/aks-preview/HISTORY.md

```diff
@@ -3,37 +3,59 @@
 Release History
 ===============
 
+0.5.59
+++++++
+
+* Add support for managed cluster snapshot commands and modify current nodepool snapshot commands
```

Comment:
Maybe it's better to remind user there's a breaking change. What used to be the `aks snapshot` command is now the `aks nodepool snapshot ` command.

However, this may also confuse users more...

Reply:
This is a good suggestion!  @zqingqing1 Please take a look~

Reply:
made the change